Sources/TensorFlow/Core/LazyTensorOperation.swift

```diff
@@ -153,6 +153,9 @@ class LazyTensorOperation: TensorOperation {
         case ConstTensor(TFETensorHandle)
         case TensorDataTypeValue(TensorDataType)
         case TensorFunctionPointer(_TensorFunctionPointer)
+        case TensorDataTypeArray([TensorDataType])
```

Comment:
Enum cases should use camel case. Didn't notice it earlier.

Reply:
Will send a follow up PR for this.